pkg/stanza/docs/operators/recombine.md

```diff
@@ -141,3 +141,146 @@ The following logs will be output:
   },
 ]
 ```
+
+#### Example configurations with `max_unmatched_batch_size`
+
+##### `max_unmatched_batch_size=0`
```

Comment:
```suggestion
##### `max_unmatched_batch_size` set to `0`
```

It sounds better for me when I read it 😅 

And the same for all cases

Reply:
I changed it :)